BraveWallet/Crypto/BuySendSwap/AccountPicker.swift

```diff
@@ -123,6 +123,13 @@ struct AccountPicker: View {
             }) {
               AccountView(address: account.address, name: account.name)
             }
+            .contextMenu {
+              Button(action: {
```

Comment:
this could be created by `copyAccountAddressButton` too in theory, but requires some code reorganization, not sure if worth to generalize such small ui piece

Reply:
Yeah I thought about it but decided its just a button so wasn't worth pulling it up into its own public class